components/navigation/Navigation.js

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components"
+
+const StyledDiv = styled.div`
+position: absolute;
+bottom: 0;
+text-align: center;
+width: 100%
+`
+
+
+export default function Navigation(){
+    return (
+        <StyledDiv>Placeholder For Navigation Elements</StyledDiv>
```

Comment:
you should not have a whole component that is only a placeholder. If a component serves no purpose yet, please remove it.